pytorch_lightning/loggers/wandb.py

```diff
@@ -159,12 +162,14 @@ def experiment(self) -> Run:
                 **self._kwargs
             ) if wandb.run is None else wandb.run
 
-            # offset logging step when resuming a run
-            self._step_offset = self._experiment.step
-
             # save checkpoints in wandb dir to upload on W&B servers
             if self._save_dir is None:
                 self._save_dir = self._experiment.dir
+
+            # define default x-axis (for latest wandb versions)
+            if getattr(self._experiment, "define_metric", None):
+                self._experiment.define_metric("*", step_metric='train/step', step_sync=True)
```

Comment:
```suggestion
                self._experiment.define_metric("*", step_metric=trainer/global_step', step_sync=True)
```

Reply:
Just to make sure I understand:
* you prefer to call the metric `global_step` vs `step`
* you want to put it in a `trainer` group vs as part of the `train` metrics group (the section before `/` automatically groups metrics in wandb)

Note: Later I want to create automatically `train` and `eval` groups by renaming metrics (or creating aliases): ie `train_loss` becomes `train/loss`